src/Components/Components/src/Rendering/HtmlRenderer.cs

```diff
@@ -22,6 +22,8 @@ public class HtmlRenderer : Renderer
             "area", "base", "br", "col", "embed", "hr", "img", "input", "link", "meta", "param", "source", "track", "wbr"
         };
 
+        private static readonly Task CancelledRenderTask = Task.FromCanceled(new CancellationToken(canceled: true));
```

Comment:
Is it 'Canceled' or 'cancelled'? Apparently both are correct. @rynowak will like the first one more, while @SteveSandersonMS will prefer the second.

Reply:
However we spell it elsewhere. From memory, I think we go with a single 'l' for everything apart from in the word "cancellation". So, should be:

```suggestion
        private static readonly Task CanceledRenderTask = Task.FromCanceled(new CancellationToken(canceled: true));
```